motionplan/motionPlanner.go

```diff
@@ -441,7 +441,7 @@ IK:
 	for k := range solutions {
 		keys = append(keys, k)
 	}
-	sort.Float64s(keys)
+	slices.Sort(keys)
```

Comment:
The docstring on `sort.Float64s` says that we should prefer this over the one we've been using since its faster.  Since this is a critical section of code I thought that it might be a useful change to make.